templates/config.fish

```diff
@@ -6,8 +6,8 @@ set -g OMF_CONFIG {{OMF_CONFIG}}
 
 ### Configuration required to load oh-my-fish ###
 # Note: Only add configurations that are required to be set before oh-my-fish is loaded.
```

Comment:
We should actually use `before.init.fish` instead of adding the configuration here.